packages/staking/src/executors_registry.rs

```diff
@@ -48,8 +51,15 @@ pub mod data_request_executors {
 
         Ok(Response::new()
             .add_attribute("action", "register_data_request_executor")
-            .add_attribute("executor", sender)
-            .add_attribute("p2p_multi_address", p2p_multi_address.unwrap_or_default()))
+            .add_event(
+                Event::new("seda-data-request-executor").add_attributes(vec![
```

Comment:
I believe this better be `seda-register-data-request-executor` for clarity and it's also more consistent with `seda-unregister-data-request-executor`